core/transport/src/grpc/file_system_master.proto

```diff
@@ -64,6 +64,7 @@ message OpenFilePOptions {
   optional ReadPType readType = 1;
   optional int32 maxUfsReadConcurrency = 2;
   optional FileSystemMasterCommonPOptions commonOptions = 3;
+  optional bool updateLastAccessTime = 4 [default = true];
```

Comment:
Should this be a common option? Is last access time updated on other calls like getStatus?

Reply:
lastAccessTime is only updated on content access(e.g. read, create, list children). It is not updated on other `getStatus` calls.